mmv1/third_party/terraform/utils/common_operation.go

```diff
@@ -111,7 +111,7 @@ func CommonRefreshFunc(w Waiter) resource.StateRefreshFunc {
 		op, err := w.QueryOp()
 		if err != nil {
 			// Retry 404 when getting operation (not resource state)
-			if isRetryableError(err, isNotFoundRetryableError("GET operation"), isOperationReadQuotaError) {
+			if isRetryableError(err, isNotFoundRetryableError("GET operation")) {
```

Comment:
I don't believe this is safe to remove- it looks for `Quota exceeded for quota group` vs the new rule which looks for `Quota exceeded for quota metric`

Reply:
Yes, this is intentional. The GCE API used to return an error message "Quota exceeded for quota group..." but this error message has now been modified to be "Quota exceeded for quota metric...". If you think there are other GCP APIs that still return an HTTP 403 with an error message of "Quota exceeded for quota group..." I can bring it back, please let me know.

Reply:
Do you happen to have a doc or something to confirm the change? That makes sense, given the change has been made.

Reply:
Yes, please see https://docs.google.com/document/d/1gaCawbQElzOwZkeaNqwhL4EKPLQtm_Ef13D3WcdO2BY/edit and https://goto.google.com/gce-api-superquota-migration